src/main/java/com/uid2/shared/store/reader/RotatingClientKeyProvider.java

```diff
@@ -93,4 +95,12 @@ public CloudPath getMetadataPath() {
     public IAuthorizable get(String key) {
         return getClientKey(key);
     }
+
+    public ClientKey getOldestClientKey(int siteId) {
```

Comment:
Should this be a method on the snapshot instead? This is not really specific to the rotating provider.

Reply:
Moved `getOldestClientKey()` to _ClientKeySnapshot_.